src/main/java/uk/gov/hmcts/ccd/data/definition/CachedCaseDefinitionRepository.java

```diff
@@ -49,6 +47,18 @@ public UserRole getUserRoleClassifications(String userRole) {
         return userRoleClassifications.computeIfAbsent(userRole, caseDefinitionRepository::getUserRoleClassifications);
     }
 
+    @Override
+    public List<UserRole> getClassificationsForUserRoleList(List<String> userRoles) {
+        final List<String> missingRoles = userRoles.stream()
+            .filter(role -> !userRoleClassifications.containsKey(role))
+            .collect(Collectors.toList());
+        if (!missingRoles.isEmpty()) {
+            final List<UserRole> missingClassifications = caseDefinitionRepository.getClassificationsForUserRoleList(missingRoles);
+            missingClassifications.forEach(userRole -> userRoleClassifications.putIfAbsent(userRole.getRole(), userRole));
```

Comment:
can we rename userRole to classification?

Reply:
done